SimG4CMS/Forward/interface/MtdSD.h

```diff
@@ -0,0 +1,140 @@
+#ifndef SimG4CMSForward_MtdSD_h
+#define SimG4CMSForward_MtdSD_h
+
+#include "DetectorDescription/Core/interface/DDsvalues.h"
+#include "SimG4Core/Notification/interface/Observer.h"
+#include "SimG4Core/SensitiveDetector/interface/SensitiveTkDetector.h"
+
+#include "SimG4Core/Notification/interface/BeginOfJob.h"
+#include "SimG4Core/Notification/interface/BeginOfRun.h"
+#include "SimG4Core/Notification/interface/BeginOfEvent.h"
+#include "SimG4Core/Notification/interface/EndOfEvent.h"
+
+#include "SimG4CMS/Forward/interface/BscG4Hit.h"
+#include "SimG4CMS/Forward/interface/BscG4HitCollection.h"
+#include "Geometry/MTDCommonData/interface/MTDNumberingScheme.h"
+  
+#include "G4Step.hh"
+#include "G4StepPoint.hh"
+#include "G4Track.hh"
+#include "G4VPhysicalVolume.hh"
+
+#include <string>
+
+class TrackingSlaveSD;
+class TrackInformation;
+class SimTrackManager;
+class TrackingSlaveSD;
+class UpdatablePSimHit;
+class G4ProcessTypeEnumerator;
+class G4TrackToParticleID;
+class MTDBaseNumber;
+
+//-------------------------------------------------------------------
+
+class MtdSD : public SensitiveTkDetector,
+                    public Observer<const BeginOfJob *>,
+                    public Observer<const BeginOfRun *>,
+                    public Observer<const BeginOfEvent*>,
+                    public Observer<const EndOfEvent*> {
+
+public:
+  
+  MtdSD(const std::string&, const DDCompactView &, const SensitiveDetectorCatalog &, 
+	      edm::ParameterSet const &, const SimTrackManager* );
+
+  ~MtdSD() override;
+  
+  bool     ProcessHits(G4Step *,G4TouchableHistory *) override;
```

Comment:
CMS code "rules" suggest using lower case initials for class members and methods (several such cases in this newly added class)

Reply:
thanks, you are correct. Anyway this is the pattern, or "style" if you prefer, of all the Geant4 SD codes in SimG4CMS. I think it would make sense a general cleaning in case